recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "gensim" %}
+{% set version = "2.2.0" %}
+{% set sha256 = "eb099de1e50447c42e168a1a99de4721923688afc71b12fe522f79687a4fbb13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - cython
+    - scipy >=0.7.0
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+  run:
+    - python
+    - numpy x.x
+    - cython
+    - scipy >=0.7.0
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+test:
+  requires:
+    - nose
```

Comment:
Its probably sufficient initially to have

```test:
    imports:
        - gensim
        - gensim.foo
```
for all the standard importable modules